src/transformers/trainer_callback.py

```diff
@@ -475,3 +476,62 @@ def on_log(self, args, state, control, logs=None, **kwargs):
         _ = logs.pop("total_flos", None)
         if state.is_local_process_zero:
             print(logs)
+
+
+class EarlyStoppingCallback(TrainerCallback):
+    """
+    A :class:`~transformers.TrainerCallback` that handles early stopping.
+
+    Args:
+       early_stopping_patience (:obj:`int`):
+            Use with :obj:`metric_for_best_model` to stop training when the specified metric worsens for
+            :obj:`early_stopping_patience` evaluation calls.
+       early_stopping_threshold(:obj:`float`, `optional`):
+            Use with TrainingAruments :obj:`metric_for_best_model` and :obj:`early_stopping_patience` to denote how
```

Comment:
Small typo here: `TrainingAruments` -> `TrainingArguments`

Reply:
Ah, thanks for catching that! I'm embarrassingly typo prone :sweat_smile: 
```suggestion
            Use with TrainingArguments :obj:`metric_for_best_model` and :obj:`early_stopping_patience` to denote how
```